app/controllers/ems_cloud_controller.rb

```diff
@@ -235,6 +239,9 @@ def build_credentials(ems)
       amqp_password = params[:amqp_password] ? params[:amqp_password] : ems.authentication_password(:amqp)
       creds[:amqp] = {:userid => params[:amqp_userid], :password => amqp_password}
     end
+    if ems.supports_authentication?(:service_account) && params[:service_account_json]
+      creds[:service_account] = {:json => params[:service_account_json], :userid => "_"}
```

Comment:
Can the key name here be `:service_account_json` as well?  Later in [the authentication mixin](https://github.com/ManageIQ/manageiq/pull/5208/files#diff-f9cace1fe4fdcec66f962b23f2110fd6R162) this context gets lost.